---
title: research
nav:
  order: 3
  tooltip: Research description
---

# <i class="fas fa-microscope"></i>Research Summary

**We are a bioinformatics lab pursuing a fundamental biological question, that is, how a cell identity is established in development, maintained in a healthy body, and dysregulated in diseases.** We develop bioinformatics methods to understand genetic and epigenetic mechanisms in cell identity regulation. Different cell types in a healthy body share the same genetic sequence. Difference in identity of these cell types is determined by epigenetic regulation of gene expression. Meanwhile, many genetic mutations in germ line or somatic cells were found to affect epigenetic factors, and can cause disease due to cell identity dysregulation. (1) To study cell identity regulation in development, we are interested in cardiovascular endothelial lineage specification. (2) To study cell identity dysregulation in diseases, the lab is focused on cancers and cardiovascular diseases.

{%
  include figure.html
  image="images/Cell_Identity.png"
  width="100%"
  height="100%"
%}


**Major research directions in the lab:**

- Develop new bioinformatics techniques to interpret high throughput genomic data.

- Single cell, spatial genomics, Hi-C, ChIP-seq, ATAC-seq, RNA-seq and other additional techniques to study epigenetic regulation of transcription by 3D chromatin conformation, chromatin modifications, and chromatin-binding proteins.

- High throughput profiling of RNA methylation, RNA binding proteins, and ribosomes to study post transcriptional regulation of gene expression.

- Molecular, cellular, animal, and clinical models to understand mechanisms in diseases with a focus on heart failure and prostate cancer.

**The most important discovery in the lab is that the cell identity genes as a unique group are distinct from other genes in the molecular mechanisms to regulate their expression and mutation patterns** (Chen et al, Nature Genetics 2015; Xia et al, Nature Communications 2020a; Zhao et al, Nature Communications 2020b; Wang et al, GPB 2021; Li et al, Nucleic Acids Research, 2023; Lv et al, Nature Communications, 2023). Specifically, we find that due to broad enrichment of transcription factor binding motifs at cell identity gene loci, these genes are regulated by unique epigenetic mechanisms such as the Broad H3K4me3 to maintain a stable transcriptional elongation status. We further found that these genes show frequent co-transcriptional m6A modifications on their RNAs to specify fast decay of the RNAs in the cells. Therefore, genetic elements determined that cell identity genes are regulated by epigenetic mechanisms that promote fast turnover of the gene expression. In contrast, most other active genes in the cell, such as housekeeping genes, undergo slow  expression turnover. This finding builds a knowledge foundation for us to develop bioinformatics technology for discovery of cell identity regulators and disease genes. Our findings further provide insights into the molecular regulation of normal cell identities in development (Gu et al, Science 2019), molecular regulation of genetic variations (Chen et al, NAR 2008; Yu et al, Nature 2018; Hu et al, Genes & Development 2014), and dysregulation of cell identity in diseases, e.g., in limb ischemia (Meng, et al, Circulation 2020), atherosclerosis (Circulation Research 2023), and cancers (Yi et al, Nature Cell Biology 2021; Zhao et al, Nature Communications 2020b; Dhar et al, Molecular Cell 2018). The techniques we developed has been proved to successfully recapture known disease genes such as cancer driver genes, and further, overcome critical constraint on current technologies by successfully identifying many disease genes that were not detectable by conventional mutation analysis (Chen et al, Nature Genetics 2015; Nature Communications 2020b). These disease genes led to new therapeutic targets and diagnostic markers as have been verified in cellular, animal, and preclinical models by our collaborators (Zhu et al, Nature Communications 2018; Liu et al, IJC 2018; Zhu et al, Oncogene 2019) and thus, will benefit numerous cancer patients.


