---
title: tools
nav:
  order: 3
  tooltip: Software, datasets, and more
---

# <i class="fas fa-tools"></i>Software and Database


**[SCIG](https://github.com/kaifuchenlab/SCIG) (Nucleic Acids Research, 2025):**
SCIG: A AI tool for Uncovering the Cell Identity Genes (CIGs) and their master transcription factors at single-cell levels, utilizing genetic sequence and RNA expression profiles.

**[NanoNm](https://github.com/kaifuchenlab/NanoNm) (Molecular Cell, 2024):**
A machine learning model to detect the RNA modification 2'-O-methylation (Nm) in Nanopore direct RNA sequencing data.

**[MCCP](http://rc-cbp.tch.harvard.edu/mccp/) [(BioRxiv, 2022)](https://www.biorxiv.org/content/10.1101/2022.05.30.494067v1):**
A knowledgebase and web portal for accessing the landscape of metabolite-mediated cell-cell communications.

**[MEBOCOST](https://github.com/zhengrongbin/MEBOCOST) [(Nucleic Acids Research, Accepted)](https://www.biorxiv.org/content/10.1101/2022.05.30.494067v1):**
A computational technology to enable detection of cell-cell communications mediated by metabolites and their sensor proteins (receptor, transporter, etc) based on single-cell RNA-Seq data.

**[StripeDiff](https://github.com/GuangyWang/stripeDiff) (Science Advances, 2022):**
A statistical method and software to identify differential stripes between two Hi-C samples.

**[MACMIC](https://github.com/bxia888/MACMIC) (Genomics, Proteomics and Bioinformatics, 2021):**
A statistical method to define epigenomic features that have maximal co-localization but minimal correlation across the genome.

**[TADsplimer](https://github.com/GuangyWang/TADsplimer) (Genome Biology, 2020):** 
A bioinformatic toolkit to identify TAD splits and mergers between Hi-C samples.

**CEFCIG at [GitHub](https://github.com/bxia888/CEFCIG) or [GoogleSite](https://sites.google.com/site/cellidentitygene/)(Nature Communications, 2020):** 
A Computational Epigenetic Framework for Cell Identity Gene discovery.
- GridGO: a grid-based genetic algorithm for automatic optimization of epigenetic signature analysis on the basis of biological significance.
- CIGdiscover: a logistic regression model for cell identity genes discovery
- CIGnet: a novel network method to uncover master transcription factors for cell identity genes in an unbiased, knowledge-guided, and quantitatively testable manner.
- CIGDB: A Cell Identity Gene Database.

**[CAGRE](https://github.com/jielv/CAGRE) (Science, 2019):**
A bioinformatic tool for Commulative Analysis of Genomic Region Enrichment.

**[FAID](https://github.com/fagisX/FAID) (Nature, 2018):**
A bioinformatic tool for Feature Analysis of Inserted DNA in a genome.


**[DANPOS2](https://sites.google.com/site/danposdoc) (Nature Genetics, 2015):**
a set of 8 bioinformatics tools, including:
- Dpos: comparative analysis of MNase-Seq data between samples.
- Dpeak: comparative analysis ChIP-Seq data between samples.
- Dregion: analyzing broad epigenetic domains, e.g., super enhancers.
- Stat: statistical analysis of positions, peaks, or regions defined by DANPOS.
- Selector: mapping positions, peaks, or regions to any given genomic features such as genes.
- Profile: analyzing epigenetic signal at any given genomic regions, e.g., genes, enhancers.
- Wiq: a tool for genome wide quantile normalization of epigenetic data.
- Wig2Wiq: a tool for converting wiggle format file to wiq format.

**[DANPOS](https://sites.google.com/site/danposdoc) (Genome Research, 2013):**
A tool for Dynamic Analysis of Nucleosome and Protein Occupancy by Sequencing

**SpikeSCN (Genes & Development, 2013):**
Genome structure and copy number variation analysis based on genome sequencing with spike-in control.

**BarVar (Cell reports, 2013):**
A bioinformatics pipeline to screen for drug-resistant genetic variations from barcode sequencing data.

**NucHMM (GPB, 2010):**
A Hidden Markov Model for nucleosome positioning prediction based on DNA sequence feature.

**GenomePSD (Nucleic acids research, 2008):**
A Power Spectrum Density analysis tool for genome sequence.
